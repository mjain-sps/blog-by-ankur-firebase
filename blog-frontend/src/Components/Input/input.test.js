import React from "react";
import { render } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";
import InputComponent from "./input.textbox.component";
describe("Input component", () => {
  test("check input snapshot", () => {
    const dummyProps = {
      placeholder: "dummy input",
      value: "Madhur Jain",
      onChange: jest.fn(),
      name: "email",
      maxLength: 16,
      type: "email",
      id: "email-id",
    };
    const { container } = render(<InputComponent {...dummyProps} />);
    expect(container).toMatchSnapshot();
  });
});
