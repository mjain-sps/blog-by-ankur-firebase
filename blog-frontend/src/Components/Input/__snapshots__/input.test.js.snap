// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input component check input snapshot 1`] = `
<div>
  <input
    class="sc-bdnxRM cQVKlz"
    id="email-id"
    maxlength="16"
    name="email"
    placeholder="dummy input"
    type="email"
    value="Madhur Jain"
  />
</div>
`;
